README.md

```diff
@@ -28,15 +28,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. Describe the biggest difference between `.forEach` & `.map`.
+1. Describe the biggest difference between `.forEach` & `.map`. forEach iterates through the whole array and logs out each item in the array line by line. .map creates a new array of the specified items you want. example. If you want just an array of first names...you can map throught the array and do that.
 
-2. What is the difference between a function and a method?
+2. What is the difference between a function and a method? A function is a stand alone item in javascript there are different types of ways you can use functions and different syntaxes like arrow functions. But methods are function that are on objects. example... this.speak = function(){}.
 
-3. What is closure?
+3. What is closure? A closure is a function that can access variables from outside of the function.
```

Comment:
Close, but a closure is when a child function can access variables its parent function.